qemu/tests/secure_execution.py

```diff
@@ -0,0 +1,44 @@
+from virttest import error_context
+from virttest import virt_vm
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Secured guest boot test:
+    1) Log into a secure guest
+    2) Check if there's error messagein
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    timeout = float(params.get("login_timeout", 240))
+    serial_login = params.get("serial_login", "no") == "yes"
+    vm = env.get_vm(params["main_vm"])
+    error_context.context("Try to log into guest '%s'." % vm.name,
+                          test.log.info)
+    error_check = params.get("error_check", "no")
+    if params.get("start_vm") == "no" and "no" in error_check:
```

Comment:
Same above, use get_boolean to check if we need error_check.

Reply:
the error_check here is a param in cfg file, separate valid and invalid scenarios

Reply:
updated